### DISCUSSION TITLE
About ceph cluster maintenance

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#12288] [Q&A]
What processing should be done before the ceph cluster is powered off, and what should be paid attention to?

**Comment by travisn** [comment]
The Rook cluster is resilient to being restarted, even from power loss of the entire cluster. The challenge for the shutdown scenario is that you will want to scale down your application first, and the rook/ceph pods last. Otherwise, the rbd/cephfs volumes will hang if the ceph pods go down first.

**Reply by zhucan** [reply]
Only we want to moving storage node from one computer room to another, so we need to power off the ceph cluster firstly.

**Reply by travisn** [reply]
If ceph is taken down while the apps are still running, their IO will hang.
